docs/upgrade.md

```diff
@@ -6,202 +6,133 @@ sidebar_label: Upgrade
+#Upgrade to 0.7 OpenEBS Operator
 
-```
-helm upgrade -f https://openebs.github.io/charts/helm-values-0.6.0.yaml <release-name> stable/openebs
+kubectl apply -f https://openebs.github.io/charts/openebs-operator-0.7.0.yaml
 ```
 
-**Using customized operator YAML or helm chart**
+#### Install/Upgrade using helm chart (using stable/openebs, openebs-charts repo, etc.,) 
 
-Before proceeding with the upgrade, you must update your custom helm chart or YAML with 0.6 release tags and changes made in the values/templates.
+**The sample steps below will work if you have installed openebs with default values provided by stable/openebs helm chart.**
 
-Use the following as reference for the changes made in 0.6: 
+- Run `helm ls` to get the release name of openebs. 
+- Upgrade using `helm upgrade -f https://openebs.github.io/charts/helm-values-0.7.0.yaml <release-name> stable/openebs`
 
-- stable/openebs [PR#6768](https://github.com/helm/charts/pull/6768) or 
-- openebs-charts [PR#1646](https://github.com/openebs/openebs/pull/1646)
+#### Using customized operator YAML or helm chart.
 
-After updating the YAML or helm chart values, you can use the steps mentioned above to upgrade the OpenEBS operator depending upon your type of installation.
+As a first step, you must update your custom helm chart or YAML with 0.7 release tags and changes made in the values/templates. 
 
-**Step-2 Upgrade OpenEBS (old volumes) created with 0.5.3 or 0.5.4**
+You can use the following as references to know about the changes in 0.7: 
+- openebs-charts [PR#1878](https://github.com/openebs/openebs/pull/1878) as reference.
 
-Even after the OpenEBS operator has been upgraded to 0.6, the old volumes will continue to work with 0.5.3 or 0.5.4. Each of the volumes should be upgraded (one at a time) to 0.6, using the steps provided below.
+After updating the YAML or helm chart or helm chart values, you can use the above procedures to upgrade the OpenEBS Operator
 
-**Note:** There has been a change in the way OpenEBS Controller Pods communicate with the Replica Pods. Hence, it is recommended to schedule a downtime for the application using the OpenEBS PV while performing this upgrade. Also, ensure that you have taken a backup of the data before starting the following upgrade procedure.
+## Step 2: Upgrade the OpenEBS Volumes
 
-**Download the Upgrade Scripts**
+Even after the OpenEBS Operator has been upgraded to 0.7, the volumes will continue to work with older versions. Each of the volumes should be upgraded (one at a time) to 0.7, using the steps provided below. 
 
-You can get the requied scripts to your working directory using the following commands.
+*Note: Upgrade functionality is still under active development. It is hightly recommended to schedule a downtime for the application using the OpenEBS PV while performing this upgrade. Also, make sure you have taken a backup of the data before starting the below upgrade procedure.*
```

Comment:
**Note:** Upgrade
hightly = highly
Also, ensure ... starting the following upgrade procedure.